.gitattributes

```diff
@@ -0,0 +1,6 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto eol=lf
+
+# Denote all files that are truly binary and should not be modified.
+*.png binary
+*.jpg binary
```

Comment:
[It is good practice to leave eof newline](https://stackoverflow.com/a/729795)

Reply:
yes sir my lord